src/store/actions/cryptos.ts

```diff
@@ -0,0 +1,18 @@
+import {createAsyncThunk} from '@reduxjs/toolkit';
+import {API_URL} from '@env';
+
+export const fetchCrypto = createAsyncThunk(
+  'crypto/fetchCrypto',
+  async (code: string) => {
+    try {
+      const response: any = await fetch(`${API_URL}/${code}/metrics`);
+      const data = await response.json();
```

Comment:
constant data is unnecessary, you should be able to do `return await response.json() `

Reply:
the value return undefined if I make a ruturn await

Reply:
ok, keep the constant